tests.py

```diff
@@ -375,6 +393,51 @@ def testPHPMethod(self):
     # FIXME: Actually test this method sometime in the future.
     pass
 
+  def testWindowsTimezones(self):
+    if sys.platform == "win32":
```

Comment:
But there's only one line (the true Windows detection using system functions) which needs to be on Windows.  The rest of the test needs to mock all of those things, and so passes on Linux, and has the upside of increasing coverage on Travis/Coveralls.